db/migrate/20150107044854_users_without_location_cleanup.rb

```diff
@@ -0,0 +1,9 @@
+class UsersWithoutLocationCleanup < ActiveRecord::Migration
+  def up
+    User.delete_all(location: [nil,''])
```

Comment:
NP.. discuss and let me know and I ll change it accordingly.. IMO any location that isn't real would be good to identify those users later..